keras_contrib/applications/densenet.py

```diff
@@ -323,3 +467,146 @@ def __create_dense_net(nb_classes, img_input, include_top, depth=40, nb_dense_bl
         x = Dense(nb_classes, activation='softmax', W_regularizer=l2(weight_decay), b_regularizer=l2(weight_decay))(x)
 
     return x
+
+
+def __create_fcn_dense_net(nb_classes, img_input, include_top, nb_dense_block=5, growth_rate=12,
+                           nb_filter=-1, bottleneck=False, reduction=0.0, dropout_rate=None, weight_decay=1E-4,
+                           nb_layers_per_block=4, nb_upsampling_conv=128, upsampling_type='upsampling',
```

Comment:
rename to `nb_dense_block_layers` so naming matches `nb_dense_block` variable, should be consistently plural/not plural so perhaps `nb_dense_blocks`. 

I suggest the standard densenet should also accept `nb_dense_block_layers` in both integer and list form as well.